.github/workflows/CD.yaml

```diff
@@ -113,13 +116,20 @@ jobs:
       matrix:
         testGroup: [ 1,2,3,4 ]
     steps:
-      - uses: actions/checkout@v2
-      - name: Login to Github Packages
```

Comment:
das login in die ghcr ist überflüssig?

Reply:
doch - das ist nötig. Zumindest wenn REALDATA=True.
Ist jetzt wieder da.